### PR TITLE
feat: Added normalization adaptation for the lerobot v3.0 dataset

### PR DESCRIPTION
**Pull request by yigeoooo**
This is a normalization adaptation code for the lerobot v3.0 dataset format. By replacing this file with “/openpi/src/openpi/training/data-loader. py”, the lerobot v3.0 dataset format can be normalized correctly. And it still supports normalizing the dataset format of the old version lerobot v2.0.

### CODE REVIEW

**Comment by yigeoooo** [comment]
## Note: Specify the PyTorch Data Loading Path in `compute_norm_stats.py`

  One additional change is needed in `/openpi/scripts/compute_norm_stats.py`, inside the `create_torch_dataloader()` function where `_data_loader.TorchDataLoader(...)` is created, around lines 50-57.

  Add the following argument after:
 ```python
  num_batches=num_batches,
 ```
Add
 ```python
  framework="pytorch",
```
  This line tells the dataloader to output PyTorch tensors when computing norm_stats.json.

  By default, TorchDataLoader uses:

 ```python
  framework="jax"
```

If this is not explicitly specified, the batch may be converted into a JAX sharded array, while actual training uses the PyTorch path through train_pytorch.py.

Adding framework="pytorch",keeps normalization-stat computation and actual training on the same PyTorch data loading path. It also helps avoid issues such as JAX array to NumPy conversion,multi-GPU sharding problems, and unnecessary JAX GPU initialization, which can lead to excessive GPU memory usage or OOM.
